modules/motus/downloaddb/main.nf

```diff
@@ -0,0 +1,34 @@
+process MOTUS_DOWNLOADDB {
+    label 'process_low'
+    label 'error_ignore'
```

Comment:
Where is this coming from? A pipeline developer would define the error strategy at pipeline level so should not be included here (applies for the other tools too)